pages/linux/rmlint.md

```diff
@@ -0,0 +1,20 @@
+# rmlint
+
+> Extremely fast tool to remove remove duplicates and other lint from your filesystem.
+> More information: <https://rmlint.readthedocs.io/en/latest/index.html>.
+
+- Check directory for duplicates, empty files, and other lint:
+
+`rmlint {{path/to/directory}}`
+
+- Delete lint files found by an rmlint run:
+
+`./rmlint.sh`
+
+- Find all duplicates with the same base name:
```

Comment:
It may be better to be a little more explicit here:

```suggestion
- Find only duplicates that have the same filename in addition to the same contents:
```

Reply:
Applied in [`c446a4d`](https://github.com/tldr-pages/tldr/pull/9090/commits/c446a4da2b75581568aaaf8f5fd465c2ce5350ec).